cypress/integration/inventory/create-holdings-as-different-user.spec.js

```diff
@@ -5,15 +5,16 @@ import { MultiColumnListCell } from '../../../interactors';
 import permissions from '../../support/dictionary/permissions';
 import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
 import HoldingsRecordView from '../../support/fragments/inventory/holdingsRecordView';
-import getRandomPostfix from '../../support/utils/stringTools';
 import TestTypes from '../../support/dictionary/testTypes';
 import users from '../../support/fragments/users/users';
```

Comment:
Please initialize the import fragment starting from the Uppercase letter

Reply:
Done